Source/Charts/Renderers/LegendRenderer.swift

```diff
@@ -205,7 +205,7 @@ open class LegendRenderer: Renderer
         }
         
         let labelFont = legend.font
-        let labelTextColor = legend.textColor
+        var labelTextColor = legend.textColor
```

Comment:
this should be `let legendTextColor...`

Reply:
@jjatie It's been a long time. I have to double-check it